# One raw Data sample from WG035 F/T input via WG006 (gripper MCB).
int16[] data
uint16  vhalf